woocommerce/woocommerce-android
# Free Trial: Redesign app menu to use row button sections

Summary
==========
Fix issue #8809 and #8811 by adjusting the More Menu section of the app to abandon a grid style and adopt a list one.

⚠️ The Figma designs uses an very `iOS` styled approach, so I decided to deviate a little bit from it using the Android card list style. 

Also, the view header is not matching the Figma designs because since that new header adds a Store Plan badge, it will require some different code work that I plan to introduce through #8810.

Screenshots
==========
| Figma design  | Actual implementation |
| ------------- | ------------- |
| ![iOS-Menu - new](https://user-images.githubusercontent.com/5920403/231900738-370821e4-265e-4d68-ad29-bcbc1e5a82bd.png) | ![Screenshot_20230413_193037](https://user-images.githubusercontent.com/5920403/231900727-ed11ff19-d3b0-440f-91db-b0745ae1ec15.png) |

How to Test
==========
1. Open the app and go to the `More menu` section. 
2. Verify that the app now uses a new list style containing icons, title and description.
3. Verify that the settings is now accessible from a menu button card instead of the top-right corner button
4. Verify that all buttons work the same way they did before, considering their visibility rules and which view they open

Update release notes:
